stats/stats_central.php

```diff
@@ -242,6 +244,21 @@ function count_books_in_state($state, $clauses = "")
 
 $table->end();
 
+// XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+// Miscellaneous Graphs
+
+$table = new ThemedTable(
+  2,
+  _("Miscellaneous Graphs")
+);
+
+$table->row(
+  "<a href='equilibria.php'>" . _("Pages Saved per Day") . "</a>",
```

Comment:
Most people would recognize "Equilibria" -- maybe put that in parens after the link? Or put each one on a separate line with a short description as with the section above.

Reply:
I went with the paren option. I hope that is acceptable.

Reply:
Works for me -- I just wanted something that would be recognizable.